src/HotChocolate/Core/src/Types/Types/Descriptors/Contracts/IArgumentDescriptor.cs

```diff
@@ -40,4 +40,6 @@ IArgumentDescriptor Directive<T>()
         where T : class, new();
 
     IArgumentDescriptor Directive(NameString name, params ArgumentNode[] arguments);
+
+    IArgumentDescriptor ConfigureContextData(Action<ExtensionData> configure);
```

Comment:
I have reversed this specific change since its not necessary to get the feature in. We should be careful in introducing new things to the descriptors.

Reply:
@michaelstaib  I align it with the IObjectField Descriptor